test/extended/openstack/volumes.go

```diff
@@ -180,6 +184,68 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack pla
 				return nil
 			}, "60s", "10s").Should(o.BeNil())
 		})
+
+		g.It("should create a manila share when using manila storage class", func() {
+			var err error
+			manilaSc := FindStorageClassByProvider(oc, "manila.csi.openstack.org")
+
+			// Skip if Manila Storage class is not defined
+			if manilaSc == nil {
+				e2eskipper.Skipf("No StorageClass with manila.csi.openstack.org provisioner")
+			}
+
+			ns := oc.Namespace()
+			pvc := CreatePVC(ctx, clientSet, "manila-pvc", ns, manilaSc.Name, "1Gi")
+			fileContent := "hello"
+
+			shareClient, err = client("sharev2")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			// Make sure a Manila share was created with the same name as the PVC
+			err = waitPvcVolume(ctx, clientSet, pvc.Name, ns)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			manilaShares, err := GetSharesFromName(shareClient, pvc.Spec.VolumeName)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			//			_, err := shares.Get(shareClient, pvc.Spec.VolumeName).Extract()
+			//			manilaShares, err := GetSharesFromName(shareClient, pvc.Spec.VolumeName)
+			//			o.Expect(err).NotTo(o.HaveOccurred())
+			//			e2e.Logf("PVC: %v, Shares: %v", pvc.Spec.VolumeName, manilaShares)
+			o.Expect(len(manilaShares)).To(o.Equal(1))
+			shareID := manilaShares[0].ID
+			_, err = shares.Get(shareClient, shareID).Extract()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// Creating a deployment with the volumes attached
+			g.By("Creating Openshift deployment")
+			labels := map[string]string{"app": "manila-test-dep"}
+
+			dep := NewDeploymentOpts()
+			dep.Name = "manila-test-dep"
+			dep.Labels = labels
+			volOpt := volumeOption{
+				Name:      "data-volume",
+				PvcName:   pvc.Name,
+				MountPath: "data",
+			}
+			dep.Volumes = append(dep.Volumes, volOpt)
+
+			//testDeployment := createTestDeployment("manila-test-dep", labels, 2, "TCP", 8080, "data-volume", pvc.Name, "data")
```

Comment:
Please remove. Maybe you can add a text line explaining the defaults

Reply:
Removed. Thanks. I modified the By method to included replicas and volumes attached